package da.protobuf;

import "math.proto";

message GameData
{
	optional bool debug = 1;
	optional string map_name = 2;
	optional string server_name = 3;
	optional int32 timestamp = 4;
	optional PlayerPositions positions = 5;
	optional bool cheats = 6;
	optional PlayerPositions kills = 7;
	optional PlayerPositions deaths = 8;
	optional int32 connections = 9;
	optional bool teamplay = 10;

	repeated string characters_chosen = 16;
	repeated int32 weapons_chosen = 17;
	repeated int32 skills_chosen = 18;
}

message PlayerPositions
{
	repeated Vector position = 1;
}
